verification/spec/connection-handshake/ch.tla

```diff
@@ -0,0 +1,188 @@
+--------------------------------- MODULE CH ---------------------------------
+
+
+VARIABLES
+    parties,            \* The state of each party.
+    inboundDatagrams    \* For each party, a buffer with any incoming datagrams. 
```

Comment:
Maybe better to call it inMsgs, as what they exchange is not really datagram (with respect to ICS03).